scripts/fast-reboot

```diff
@@ -337,6 +342,12 @@ if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
     # into /host/fast-reboot
     mkdir -p /host/fast-reboot
     /usr/bin/fast-reboot-dump.py -t /host/fast-reboot
+    FAST_REBOOT_DUMP_RC=$?
```

Comment:
I don't think you will have a chance to get to here in case failure because we are running this script with '-e' option, when fast-reboot-dump.py fails, it will trigger the error exit.

There is a complicated way to achieve what you want:

https://github.com/Azure/sonic-utilities/blob/master/scripts/fast-reboot#L153

Reply:
I didn't test this, but conceptually, there is a simpler format for your purpose:

FAST_REBOOT_DUMP_RC=0
/usr/bin/fast-reboot-dump.py -t /host/fast-reboot || FAST_REBOOT_DUMP_RC=$?
if [[ FAST_REBOOT_DUMP_RC -ne 0 ]]; then
...
fi